google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/model/ErrorProtoJsonProtocol.scala

```diff
@@ -52,8 +52,8 @@ object ErrorProtoJsonProtocol {
    * @param message A human-readable description of the error
    * @return an [[ErrorProto]]
    */
-  def createErrorProto(reason: String, location: util.Optional[String], message: String) =
-    ErrorProto(reason, location.asScala, message)
+  def createErrorProto(reason: util.Optional[String], location: util.Optional[String], message: util.Optional[String]) =
```

Comment:
Well usually it would be `ErrorProto.create` but in this case it is `ErrorJsonProtocol.createErrorProto`. Actually, I've been having doubts about keeping all the model classes inside `JsonProtocol` objects. It's ok for Scala but it is much more difficult to use from Java. I can fix this in a quick followup PR?

Reply:
Follow up PR is fine :+1:

Reply:
Tracking in #2625